packages/sign-client/src/controllers/engine.ts

```diff
@@ -349,7 +352,7 @@ export class Engine extends IEngine {
   public request: IEngine["request"] = async <T>(params: EngineTypes.RequestParams) => {
     await this.isInitialized();
     await this.isValidRequest(params);
-    const { chainId, request, topic, expiry } = params;
+    const { chainId, request, topic, expiry = FIVE_MINUTES } = params;
```

Comment:
Should be using mapped constant rather than a plain duration constant?

```suggestion
    const { chainId, request, topic, expiry = ENGINE_RPC_OPTS.wc_sessionRequest.req.ttl } = params;
```